index.js

```diff
@@ -0,0 +1,15 @@
+/*eslint-disable*/
+import BooksdataBase from './Modules/library.js';
+import pageNav from './Modules/navigation.js';
+
+const date = document.getElementById('date');
+
+window.setInterval(() => {
+  // eslint-disable-next-line no-undef
+  date.innerHTML = `<p>${luxon.DateTime.local().toLocaleString(luxon.DateTime.DATETIME_MED_WITH_SECONDS)
```

Comment:
- [ ] From the requirements of this project, the current date should be displayed below the navigation bar. Please also make sure to address this issue.

Reply:
its showing on my device please check yours